pkg/types/ovirt/config.go

```diff
@@ -0,0 +1,75 @@
+package ovirt
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+
```

Comment:
this line should be removed so that both of these external packages are in a single block